tests/server/tso/local_tso_test.go

```diff
@@ -211,20 +176,22 @@ func (s *testLocalTSOSuite) TestLocalTSOAfterMemberChanged(c *C) {
 			for j := 0; j < 30; j++ {
 				for _, dcLocation := range dcLocationConfig {
 					req := &pdpb.TsoRequest{
-						Header:     testutil.NewRequestHeader(cluster.GetCluster().GetId()),
+						Header:     testutil.NewRequestHeader(leaderServer.GetClusterID()),
 						Count:      tsoCount,
 						DcLocation: dcLocation,
 					}
 					ctx, cancel := context.WithCancel(context.Background())
 					ctx = grpcutil.BuildForwardContext(ctx, cluster.GetServer(leaderServer.GetAllocatorLeader(dcLocation).GetName()).GetAddr())
-					ts := testGetLocalTimestamp(c, ctx, dcClientMap[dcLocation], req)
+					ts := testGetTimestamp(c, ctx, dcClientMap[dcLocation], req)
 					cancel()
 					lastTS := lastList[dcLocation]
 					// Check whether the TSO fallbacks
 					c.Assert(tsoutil.CompareTimestamp(ts, lastTS), Equals, 1)
-					// Because we have a Global TSO synchronization, even though the system time
-					// of the PD nodes in dc-4 is slower, its TSO will still be big enough.
-					c.Assert(tsoutil.CompareTimestamp(ts, globalTS), Equals, 1)
+					if globalTS != nil {
```

Comment:
Parameter `globalTS` is too specified, maybe we can use a name like `preTS` or `previousTS` to indicate it's a TS from the previous procedure.